_posts/2022-10-03-the-definitive-guide-2.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Refresh of Trino: The Definitive Guide"
+author: Manfred Moser, Martin Traverso, Matt Fuller
+excerpt_separator: <!--more-->
+---
+
+It was time for a refresh. Only a little while ago, in April 2021, we announced
+the [Trino version of our definitive guide]({{ site.url }}{% post_url
+2021-04-21-the-definitive-guide %}). But again Trino as a project and community
+has continued to innovate and grow. Numerous smaller and larger details changed,
+and the examples and resources really needed to be fixed.
```

Comment:
```suggestion
and the examples and resources needed to be fixed.
```
I'm formally declaring war on unnecessary adverbs

Reply:
Nice .. thanks for catching that mess